### ISSUE TITLE
How to get section/key in case-insensitive way?

### ISSUE THREAD

**Issue by silvioprog** [zonyitoo/rust-ini#61]
Hi.

I have a INI file with some case issues, for example:

```ini
[NFe_PB_P]
NfeConsultaProtocolo_4.00=https://nfe.sefaz.ba.gov.br/webservices/NFeConsultaProtocolo4/NFeConsultaProtocolo4.asmx
```

however, when I try to get it as following:

```rust
ini.get_from(Some("nfe_ba_p"), "nfeconsultaprotocolo_4.00");
```

it always returns `None`.

Is there any option to `Ini` work in [case-insensitive](https://en.wikipedia.org/wiki/INI_file#Case_insensitivity) way?

Thank you!

**Comment by zonyitoo** [comment]
Currently not supported. Maybe we can add a feature to add a customized hasher which works in case insensitive way

**Comment by silvioprog** [comment]
Great news! It will be really very useful, specially for Windows users.

Maybe the `UneCase` library could be used, it allows to combine with `HashMap`, e.g.: https://stackoverflow.com/a/47813878/3268398 .

**Comment by zonyitoo** [comment]
Already pushed to `master`

**Comment by silvioprog** [comment]
Well done! Thanks for implementing.

I have been trying to build it, but it raises this same CI error at build time: https://travis-ci.org/zonyitoo/rust-ini.

**Comment by zonyitoo** [comment]
Yup. Build it with nightly.

**Comment by silvioprog** [comment]
Isn't it possible to run it in stable build for production? :-/

**Comment by zonyitoo** [comment]
It should. I am trying to find a replacement.

**Comment by zonyitoo** [comment]
@silvioprog Please try with the latest commit.

**Comment by silvioprog** [comment]
Tested and worked like a charm! 🎉 Could you push this new version to crates.io? Thank you very much again! :-)

**Comment by zonyitoo** [comment]
I will need more tests. I will publish it a few days later.

**Comment by silvioprog** [comment]
Awesome! :-)

**Comment by zonyitoo** [comment]
Published v0.15.0

**Comment by silvioprog** [comment]
Thank you very much! :-)
